std_msgs/String robot_hash
